app/config/config.yml

```diff
@@ -32,6 +31,7 @@ framework:
     fragments:       ~
     http_method_override: true
     assets: ~
+        #version: SomeVersionScheme
```

Comment:
This makes sense, but hides the fact that the assets can (and should) be versioned. Not sure what is better / worse.